build.gradle

```diff
@@ -400,6 +396,17 @@ uploadArchives {
                 ' for more information.' )
             throw new Exception("Missing sonatype username or password, please see the README for properties that must be specified for uploading an archive.")}
         )
-
     }
+
+distributions {
+    spark{} //added to allow an installSparkDist target which will also build the spark jar
+}
+
+installSparkDist.dependsOn downloadGsaLibFile, sparkJar
```

Comment:
Alternatively, we could make `gradle installDist` do everything (local dist + spark jar + other things as we add them), if that is more consistent with gradle convention, and have `gradle installSpark` for just the spark jar and `gradle installLocal` (or something like that) for just the unpackaged local version.